resources/assets/sass/components/_forum.scss

```diff
@@ -0,0 +1,94 @@
+.c-forum {
+}
+
+	.c-forum__nav {
+	}
+
+		.c-forum__nav-list {
+
+			@include clear-list;
+
+			margin-bottom: 48px;
+		}
+
+			.c-forum__nav-list-item {
+
+				margin-bottom: 24px;
+			}
+
+	.c-forum__list {
+
+		@include clearfix;
+		@include clear-list;
+	}
+
+		.c-forum__list-item {
+
+			display: flex;
+			flex-flow: row wrap;
+			align-items: center;
+			padding: 9px 24px;
+			margin-bottom: 6px;
+			border-left: 2px solid $color-fill-neutral;
+
+			&:nth-child(even) {
+
+				background: $color-fill-neutral-lightest;
+				box-shadow: 0 1px 1px rgba($color-fill-neutral-light, 0.75);
+			}
+
+			&:last-child {
+
+				margin-bottom: 0;
+			}
+
+			.c-forum__list.m-compact & {
+
+				padding: 6px 0;
+				margin-bottom: 6px;
+				border: none;
+
+				&:nth-child(even) {
+
+					background: none;
+					box-shadow: none;
+				}
+			}
+		}
+
+			.c-forum__list-item-profile {
```

Comment:
According to BEM, should it not be something like

    .c-forum__list-item--profile ?